manifests/global.yaml

```diff
@@ -564,6 +564,11 @@ global:
   # we may change the default value of this option as "citadel".
   pilotCertProvider: "kubernetes"
 
+  # Configure the policy for validating JWT.
+  # Currently, two options are supported: "trustworthy-jwt" and "legacy-jwt".
+  jwtPolicy: "trustworthy-jwt"
```

Comment:
We used to have a setting `useTrustworthyJwt`, do you think it makes sense to go back to using the same setting?

If not, should we call this first and third party jwt?

Reply:
Changed to call it first and third party JWT.
Here jwtPolicy is used to express different JWT policies; the binary setting of useTrustworthyJwt makes it awkward to express different JWT policies while reducing the code readability.